samples/ImplicitFlow/AureliaApp/aurelia_project/generators/attribute.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Just delete and commit. The related commits will disappear when squashing (before merging your PR).